### PR TITLE
allow disabling daily saved exports

### PR DESCRIPTION
**Pull request by snopoke**
The impetus for this is for enterprise setups where there are exports that need to be run manually so it doesn't make sense to have them attempted every night.

image

cc @biyeun / @orangejenny for UI changes

cc @dimagi/product FYI change to daily saved export UI

### CODE REVIEW

**Comment by snopoke** [comment]
@dimagi/js-team should I be fixing those 'trailing comma' warnings?

**Comment by orangejenny** [comment]
> @dimagi/js-team should I be fixing those 'trailing comma' warnings?

yes, please

File: corehq/apps/export/templates/export/partial/export_list_controller.html [hunk 1]
```diff
@@ -144,6 +146,14 @@
                                 This export is filtered to data you don't have access to.
                                 {% endblocktrans %}
                             </div>
+                            <p class="text-warning"
+                               ng-if="export.emailedExport && !export.isAutoRebuildEnabled">
+                                <i class="fa fa-exclamation-triangle"></i>
+                                {% blocktrans %}
+                                    Automatic updates for this saved {{ export_type }} have been disable. Click 'Enable'
```
Comment (biyeun) [inline corehq/apps/export/templates/export/partial/export_list_controller.html]: disabled---forgot the d :)

File: corehq/apps/export/static/export/js/list_exports.ng.js [hunk 2]
```diff
@@ -127,6 +127,23 @@
...
+            component.savingAutoRebuildChange = true;
+            djangoRMI.toggle_saved_export_enabled_state({
+                'component': component,
+                'export': exp
+            })
+                .success(function (data) {
+                    if (data.success) {
+                        var exportType = _(exp.exportType).capitalize();
+                        var event = (exp.isAutoRebuildEnabled ? "Disable": "Enable") + " Saved Export";
+                        analytics.usage(event, exportType, "Saved");
```
[the remark this answers is not in the document]
Comment (emord) [reply corehq/apps/export/static/export/js/list_exports.ng.js]: I think we normally use `window.analytics`

**Review by emord — APPROVED** [review]
Looks good to me. the only changes suggested are text related

**Comment by snopoke** [comment]
updated

**Review by emord — APPROVED** [review]

**Review by kaapstorm — APPROVED** [review]

**Comment by snopoke** [comment]
Waiting on @dimagi/product

**Pull request merged**